config.json

```diff
@@ -488,6 +488,7 @@
         "slug": "hello-world",
         "name": "Hello World",
         "uuid": "cc96d65d-1c79-40d0-8fd2-9a6665a43b01",
+        "practices": [],
```

Comment:
@ErikSchierboom Would it be a lot of work for you to revert those changes to config.json? They will conflict with https://github.com/exercism/elixir/pull/583

Reply:
No that's quite easy. I've updated the PR.